packages/EasyBatch/src/Serializer/Serializer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyBatch\Serializer;
+
+use EonX\EasyBatch\Interfaces\SerializerInterface;
+
```

Comment:
We don't want to test it?

Reply:
I think no, it contains plain logic, just a few PHP functions.